pyproject.toml

```diff
@@ -49,6 +49,7 @@ pyrate-limiter = "~3.1.0"
 python-dateutil = "*"
 python-ulid = "^3.0.0"
 PyYAML = "^6.0.1"
+rapidfuzz = "^3.10.1"
```

Comment:
Is this still needed?

Reply:
Nevermind! I just found this description in the PR body:

> Expanded dependency management with the explicit addition of rapidfuzz (required to build SDM image).

Perfect as-is. Resolving.